spring-cloud-gcp-logging/pom.xml

```diff
@@ -27,6 +27,10 @@
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-logging-logback</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback.contrib</groupId>
+            <artifactId>logback-json-classic</artifactId>
```

Comment:
I think this should be `optional`. Nobody says that I am always going to use JSON format for logging.

Reply:
If I set the dependency to optional, everyone has to include both `spring-cloud-gcp-logging` and `logback-json-classic`. I don't think that's very user-friendly. Is it necessary to create an additional project `spring-cloud-gcp-json-logging` to have this dependency not optional?